test/TestHelpers.jl

```diff
@@ -216,6 +216,8 @@ end
     @inbounds deleteat!(parent(A), first_idx:last_idx)
 end
 
+Base.resize!(A::OffsetVector, nl::Integer) = resize!(A.parent, nl)
```

Comment:
I think I understand what you are saying here:
```julia
julia> A = OffsetArray([1,2,5,3], 4)
OffsetArray{Int64,1,Array{Int64,1}} with indices 5:8:
 1
 2
 5
 3

julia> resize!(A, 2)
2-element Array{Int64,1}:
 1
 2

julia> A
OffsetArray{Int64,1,Array{Int64,1}} with indices 5:6:
 1
 2
```
Having `indices 5:6` is consistent with the idea that `resize!` chops off the end of the array.